### PR TITLE
Some Design Updates

### PR DESCRIPTION
**Pull request by chadlagore**
# Touchups

+ Added Log Out button
+ Added new fonts and such
+ New google log in button

### CODE REVIEW

File: src/views/Home.js [hunk 1]
```diff
@@ -50,27 +50,44 @@ export class Home extends Component {
...
-                    <Text style={styles.viewEventsText}>Discover Events</Text>
-                </TouchableOpacity>
+                <View>
+                    <TouchableOpacity
+                        style={styles.viewEvents}
+                        activeOpacity={0.6}
+                        onPress={() => this.props.navigation.navigate('EventList')}>
+                        <Text style={styles.baseText}>Discover Events</Text>
+                    </TouchableOpacity>
+                    <Text style={styles.baseText}>{'\n\n'}</Text>
```
Comment (chadlagore) [inline src/views/Home.js]: I _know_ this is the wrong way to do this.

Comment (chadlagore) [reply src/views/Home.js]: 😅

File: src/views/Home.js [hunk 2]
```diff
@@ -50,27 +50,44 @@ export class Home extends Component {
...
             );
         } else {
             return (
-                <TouchableOpacity
-                    style={styles.viewEvents}
-                    activeOpacity={0.6}
-                    onPress={() => this.props.navigation.navigate('EventList')}>
-                    <Text style={styles.viewEventsText}>Discover Events</Text>
-                </TouchableOpacity>
+                <View>
```
Comment (chadlagore) [inline src/views/Home.js]: Would like to add "Hello, User" here.

File: src/views/EventFilter.js [hunk 3]
```diff
@@ -33,16 +33,22 @@ export class EventFilter extends Component {
     _hideDateTimePicker = () => this.setState({ isDateTimePickerVisible: false });
 
     _handleDatePicked = (date) => {
+        _date = Moment(date);
```
Comment (jordanschalm) [inline src/views/EventFilter.js]: If you don't put `let/const` before declarations, they're global!

File: src/components/EventCell.js [hunk 4]
```diff
@@ -53,11 +53,13 @@ const styles = StyleSheet.create({
         borderBottomColor: '#ccc'
     },
     title: {
+        fontFamily: 'Apple SD Gothic Neo',
```
Comment (jordanschalm) [inline src/components/EventCell.js]: 😲 🎨 🤺

File: src/views/Home.js [hunk 5]
```diff
@@ -50,27 +50,47 @@ export class Home extends Component {
...
+        this._renderButtons()
+    }
+
     _renderButtons () {
         if (!this.props.token) {
             return (
                 <View>
+                    <Text style={styles.titleText}>Welcome to Eventador</Text>
+                    <Text style={styles.baseText}>Let's find you something to do tonight</Text>
+                    <Text style={styles.baseText}>{'\n\n\n'}</Text>
```
Comment (jordanschalm) [inline src/views/Home.js]: 😆

Comment (jordanschalm) [reply src/views/Home.js]: If you're wondering, the standard way to do this is to add `paddingBottom` to the style.

File: src/views/Home.js [hunk 6]
```diff
@@ -50,27 +50,47 @@ export class Home extends Component {
         );
     }
 
+    _reloadComponent() {
+        this.props.setAuthToken(undefined);
+        this._renderButtons()
```
Comment (jordanschalm) [inline src/views/Home.js]: This line won't do anything ;)

File: src/views/Home.js (hunk 2 above)
Comment (agentfll) [reply src/views/Home.js]: I'll add this with my attendee list PR

File: src/views/EventFilter.js (hunk 3 above)
Comment (chadlagore) [reply src/views/EventFilter.js]: 😆

File: src/views/Home.js (hunk 5 above)
Comment (chadlagore) [reply src/views/Home.js]: ty ty 🙏

File: src/views/Home.js (hunk 2 above)
Comment (agentfll) [reply src/views/Home.js]: This should actually be possible with the currently available info--if `this.props.user.user` is set, you should be able to get at the name with `this.props.user.user.given_name` and `this.props.user.user.family_name` (those are both fields we get from Google when the user logs in, and we keep the object from Google around in our redux store)

File: src/views/Home.js (hunk 6 above)
Comment (chadlagore) [reply src/views/Home.js]: Ohhh because it's always refreshing anyways right?

File: src/views/EventFilter.js (hunk 3 above)
Comment (chadlagore) [reply src/views/EventFilter.js]: Ohhhh javascript 😅

File: src/views/Home.js (hunk 5 above)
Comment (chadlagore) [reply src/views/Home.js]: I _knew_ I was doing a bad

**Pull request merged**